tests/conftest.py

```diff
@@ -480,23 +484,31 @@ async def _make_pkls(
             for _ in range(3):
                 await socket.send_multipart([b"emptyness"])
                 await asyncio.sleep(0.1)
-        with open(filename, "rb") as f:
-            i = 0
-            while True:
-                try:
+        base, ext = os.path.splitext(filename)
+        if ext == ".gz":
+            f = gzip.open(filename, "rb")
+            _, ext = os.path.splitext(base)
+        else:
+            f = open(filename, "rb")
+        i = 0
+        while True:
+            try:
+                if "cbor" in ext:
```

Comment:
As the fixture is called `stream_pkls` maybe it is better to create a new fixture stream_cbor ?
I see the large portion of duplication, but maybe we can at one point deprecate the insecure pickle.